features/events.yml

```diff
@@ -0,0 +1,30 @@
+name: Events
```

Comment:
Yeah, a basic events system feels like a feature. You could probably do some interesting things without the DOM at all (e.g., `resize`).